packages/unit-testing/stubs/xml/Node.js

```diff
@@ -0,0 +1,358 @@
+         * @governance none
+         * @param {Object} options
+         * @param {string} options.prefix the prefix to look for; if this parameter is null, the method will return the default namespace URI if any
+         * @return {string} the associated namespace URI or null if none is found
+         *
+         * @since 2015.2
+         */
+        this.lookupNamespaceURI = function (options) { };
+
+        /**
+         * Look up the prefix associated to the given namespace URI, starting from this node. The default namespace declarations are ignored by this method.
+         * @governance none
+         * @param {Object} options
+         * @param {string} options.namespaceURI the namespace URI to look for
+         * @return {string} an associated namespace prefix if found or null if none is found; if more than one prefix are associated to the namespace prefix, the returned namespace prefix is implementation dependent
+         *
+         * @since 2015.2
+         */
+        this.lookupPrefix = function (options) { };
+
+        /**
+         * Puts all Text nodes in the full depth of the sub-tree underneath this Node, including attribute nodes, into a "normal" form
+         * where only structure (e.g., elements, comments, processing instructions, CDATA sections, and entity references) separates
+         * Text nodes, i.e., there are neither adjacent Text nodes nor empty Text nodes.
+         * @governance none
+         * @return {void}
+         *
+         * @since 2015.2
+         */
+        this.normalize = function () { };
+
+        /**
+         * Removes the child node indicated by oldChild from the list of children, and returns it.
+         * @governance none
+         * @param {Object} options
+         * @param {Node} options.oldChild the node being removed
+         * @return {Node} the node removed
+         * @throws {SuiteScriptError} SSS_XML_DOM_EXCEPTION if node cannot be removed for some reason
+         *
+         * @since 2015.2
+         */
+        this.removeChild = function (options) { };
+
+        /**
+         * Replaces the child node oldChild with newChild in the list of children, and returns the oldChild node.
+         * If the newChild is already in the tree, it is first removed.
+         * @governance none
+         * @param {Object} options
+         * @param {Node} options.newChild the new node to put in the child list
+         * @param {Node} options.oldChild the node being replaced in the list
+         * @return {Node} the node replaced
+         * @throws {SuiteScriptError} SSS_XML_DOM_EXCEPTION if node cannot be replaced for some reason
+         *
+         * @since 2015.2
+         */
+        this.replaceChild = function (options) { };
+
+        /**
+         * A map of key/value (string->Attr) pairs containing the attributes of this node (if it is an Element) or null otherwise.
+         * @name Node#attributes
+         * @type {Object}
```

Comment:
Documentations says type is string

Reply:
Not sure what's correct, leaving for now